jvb/rootfs/etc/cont-init.d/10-config

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/with-contenv bash
 
+export LOCAL_ADDRESS=$(ip route get "$(grep "$HOSTNAME" /etc/hosts|awk '{print $1}')" | head -n1 | cut -d " " -f6)
```

Comment:
This changes how we compute it. If DOCKER_HOST_ADDRESS is given, we should use the route for that IP instead, as we were doing before, I'd say.

Reply:
I just saw https://github.com/jitsi/docker-jitsi-meet/pull/630#issuecomment-708665793, so you can disregard my previous comment.

Reply:
We have found a solution which works for both of us.